### PR TITLE
Card guideline updates

### PR DESCRIPTION
**Pull request by mcarrano**
This PR updates the Card design guidelines to address the following:

* Separates Card and Card view into separate documents and moves Card view guidelines to align with Card view demo in the Demos section of the website.
* Adds Large (Marketing) card as a new card type.
* Adds guidance on border treatments
* Adds Expandable cards
* Adds guidance on selectable cards.
* Adds more usage examples for cards.

Closes: #2196 
Closes: #2288

### CODE REVIEW

**Comment by mcarrano** [comment]
@mmenestr @afialkoff Please review when you have a chance. I have split the Card guidelines into two parts. Card guidelines that cover card elements and usage and Card view which is now located in the Demos section. Here are the preview links:

https://patternfly-org-pr-2467-v4.surge.sh/v4/components/card/design-guidelines
https://patternfly-org-pr-2467-v4.surge.sh/v4/demos/card-view

Card guidelines are reworked as noted above.

@redallen a question for you. In my preview for Card View, I'm not seeing the HTML or React demos showing even though these were moved in the 2021.05 release (and are correct on the website). Do you know why?

**Comment by redallen** [comment]
@mcarrano You have to change the frontmatter to say `section: Demos`:
```
---
id: Card
section: demos
---
```

File: packages/v4/src/content/design-guidelines/components/card/card.md [hunk 1]
```diff
@@ -7,112 +7,79 @@ A **card** is a square or rectangular container that can contain any kind of con
...
-A card usually consists of 3 parts: 
+A card usually consists of 3 parts:
 
 <img src="./img/card.png" alt="3 parts of a card" width="394"/>  
 
 
 1. **Header**: Title of the information group
 2. **Body**: Brief details about the group
-3. **Footer**: Links to access more information about the group 
+3. **Footer**: Links to access more information about the group
```
Comment (mmenestr) [inline packages/v4/src/content/design-guidelines/components/card/card.md:17]: Just realized that we say the footer is for links, yet the example picture doesn't have a link haha

File: packages/v4/src/content/design-guidelines/demos/card-view/card-view.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+Use a card view to:
+- Display items in a data set that are best identified by a graphic or other visual representation.
+- Make information easier to visualize and compare.
+- Summarize a large number of objects at once within the same page.
+
+Do NOT use a card view to make a large amount of text-based content easier to scan. Use a table or list view instead.
+
+### Background colors for card views
+Use a white or a gray background for your card views.
```
Comment (afialkoff) [inline packages/v4/src/content/design-guidelines/demos/card-view/card-view.md:19]:
```suggestion
Card views can feature a white or gray background, depending on your use case.
```

File: packages/v4/src/content/design-guidelines/demos/card-view/card-view.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+- A card that opens an expandable panel on the right
+- A card view that toggles into a table or list view
+
+<img src="./img/white-background.png" alt="card view with white background" /> 
+
+**Note**: If you use a white background, apply a gray border to your card instead of a drop shadow.
+
+Use a gray background if the sole purpose of your content view is to display cards.
+
+<img src="./img/gray-background.png" alt="card view with gray background" />
```
Comment (afialkoff) [inline packages/v4/src/content/design-guidelines/demos/card-view/card-view.md:31]: In this image: Let's change "Cards" to lowercase in the page description and capitalize the "F" in "Patternfly" on the right.

File: packages/v4/src/content/design-guidelines/demos/card-view/card-view.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+### Background colors for card views
+Use a white or a gray background for your card views.
+
+Use a white background if your content view can change. Examples:
+- A card that opens an expandable panel on the right
+- A card view that toggles into a table or list view
+
+<img src="./img/white-background.png" alt="card view with white background" /> 
+
+**Note**: If you use a white background, apply a gray border to your card instead of a drop shadow.
```
Comment (afialkoff) [inline packages/v4/src/content/design-guidelines/demos/card-view/card-view.md:27]:
```suggestion
```

File: packages/v4/src/content/design-guidelines/demos/card-view/card-view.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+Do NOT use a card view to make a large amount of text-based content easier to scan. Use a table or list view instead.
+
+### Background colors for card views
+Use a white or a gray background for your card views.
+
+Use a white background if your content view can change. Examples:
+- A card that opens an expandable panel on the right
+- A card view that toggles into a table or list view
+
```
Comment (afialkoff) [inline packages/v4/src/content/design-guidelines/demos/card-view/card-view.md:24]:
```suggestion
**Note**: If you use a white background, apply a gray border to your card instead of a drop shadow.
```

Bumping this note to above the example image, that way it won't get lost between the image and the next background section.

File: packages/v4/src/content/design-guidelines/demos/card-view/card-view.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+<img src="./img/white-background.png" alt="card view with white background" /> 
+
+**Note**: If you use a white background, apply a gray border to your card instead of a drop shadow.
+
+Use a gray background if the sole purpose of your content view is to display cards.
```
Comment (afialkoff) [inline packages/v4/src/content/design-guidelines/demos/card-view/card-view.md:29]:
```suggestion
**Use a gray background** if the sole purpose of your content view is to display cards.
```

File: packages/v4/src/content/design-guidelines/demos/card-view/card-view.md [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+
+<img src="./img/gray-background.png" alt="card view with gray background" />
+
+### Card views on mobile
+Cards will size down appropriately based on the built-in CSS responsiveness behaviors. Most of the time, they’ll follow the grid layout, expanding and compressing at certain breakpoints. At the smallest mobile size, they take up almost the entire width of the screen.
+
+<img src="./img/mobile-cards.png" alt="mobile card view" width="375"/>
+
+### Card view actions
+You can customize card actions in a few ways. Here are some recommendations to consider depending on your use case:
```
Comment (afialkoff) [inline packages/v4/src/content/design-guidelines/demos/card-view/card-view.md:39]:
```suggestion
You can customize card actions in a few ways. Depending on your use case, consider customizing inline actions, multiple inline actions, global actions, or action cards within your card view.
```

File: packages/v4/src/content/design-guidelines/demos/card-view/card-view.md [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+
+#### Inline actions
+If an action can be performed on one card only, place the action in the card’s body or footer. You don’t need to include a checkbox because the actions are card-specific.
+
+- If you have a few cards with single actions, use the secondary button style.
```
Comment (afialkoff) [inline packages/v4/src/content/design-guidelines/demos/card-view/card-view.md:44]: Can we link to this button style?

File: packages/v4/src/content/design-guidelines/demos/card-view/card-view.md [hunk 9]
```diff
@@ -0,0 +1,81 @@
...
+- If you have multiple cards with single actions, use a link button or link text.
```
Comment (afialkoff) [inline packages/v4/src/content/design-guidelines/demos/card-view/card-view.md:45]: Can we link to this button style/text style?

File: packages/v4/src/content/design-guidelines/demos/card-view/card-view.md [hunk 10]
```diff
@@ -0,0 +1,81 @@
...
+
+<img src="./img/few-multiple-actions.png" alt="few cards with multiple actions" />
```
Comment (afialkoff) [inline packages/v4/src/content/design-guidelines/demos/card-view/card-view.md:47]: Can we write "Cards" in lowercase in this image's page description?

File: packages/v4/src/content/design-guidelines/demos/card-view/card-view.md [hunk 11]
```diff
@@ -0,0 +1,81 @@
...
+- If you have multiple cards with single actions, use a link button or link text.
+
+<img src="./img/few-multiple-actions.png" alt="few cards with multiple actions" />  
+
+Inline action cards are commonly used as navigation or drill-down indicators. In this case, present the user action on the card to differentiate this type of navigational card from a standard card. This way, users can easily distinguish the card types if your application includes both.   
+
+<img src="./img/inline-actions.png" alt="cards with inline actions" />
+
+#### Multiple inline actions
+If multiple actions can be performed on a single card, place those actions in a kebab. Position the kebab in the top-right corner of the header.
```
Comment (afialkoff) [inline packages/v4/src/content/design-guidelines/demos/card-view/card-view.md:54]:
```suggestion
If multiple actions can be performed on a single card, place those actions in a kebab. Position the kebab in the top-right corner of the card header.
```

File: packages/v4/src/content/design-guidelines/demos/card-view/card-view.md [hunk 12]
```diff
@@ -0,0 +1,81 @@
...
+
+<img src="./img/multiple-inline.png" alt="cards with multiple inline actions in a kebab" />
+
+#### Global actions
+If an action can be performed globally on multiple cards, or if it leads to multiple pieces of content at the same time, place the action in a toolbar above the card view. In the top-right corner of your card, add a checkbox so that users can select one or more items.
+
+<img src="./img/global-actions.png" alt="card view with global action toolbar" />
```
Comment (afialkoff) [inline packages/v4/src/content/design-guidelines/demos/card-view/card-view.md:61]: Same small capitalization change for "Cards" in this example!

File: packages/v4/src/content/design-guidelines/demos/card-view/card-view.md [hunk 13]
```diff
@@ -0,0 +1,81 @@
...
+<img src="./img/few-multiple-actions.png" alt="few cards with multiple actions" />  
+
+Inline action cards are commonly used as navigation or drill-down indicators. In this case, present the user action on the card to differentiate this type of navigational card from a standard card. This way, users can easily distinguish the card types if your application includes both.   
+
+<img src="./img/inline-actions.png" alt="cards with inline actions" />
+
+#### Multiple inline actions
+If multiple actions can be performed on a single card, place those actions in a kebab. Position the kebab in the top-right corner of the header.
+
+<img src="./img/multiple-inline.png" alt="cards with multiple inline actions in a kebab" />
```
Comment (afialkoff) [inline packages/v4/src/content/design-guidelines/demos/card-view/card-view.md:56]: Same small capitalization change (to lowercase) for "Cards" in this image's page description!

File: packages/v4/src/content/design-guidelines/demos/card-view/card-view.md [hunk 14]
```diff
@@ -0,0 +1,81 @@
...
+
+#### Global actions
+If an action can be performed globally on multiple cards, or if it leads to multiple pieces of content at the same time, place the action in a toolbar above the card view. In the top-right corner of your card, add a checkbox so that users can select one or more items.
+
+<img src="./img/global-actions.png" alt="card view with global action toolbar" />
+
+1. Upon hover, the card’s shadow expands.
+2. Global actions are placed in the toolbar.
+3. The card’s checkbox allows a user to perform an action on multiple cards. When a user selects a card, the checkbox turns blue to indicate selection.
```
Comment (afialkoff) [inline packages/v4/src/content/design-guidelines/demos/card-view/card-view.md:65]:
```suggestion
```

File: packages/v4/src/content/design-guidelines/demos/card-view/card-view.md [hunk 15]
```diff
@@ -0,0 +1,81 @@
...
+<img src="./img/inline-actions.png" alt="cards with inline actions" />
+
+#### Multiple inline actions
+If multiple actions can be performed on a single card, place those actions in a kebab. Position the kebab in the top-right corner of the header.
+
+<img src="./img/multiple-inline.png" alt="cards with multiple inline actions in a kebab" />
+
+#### Global actions
+If an action can be performed globally on multiple cards, or if it leads to multiple pieces of content at the same time, place the action in a toolbar above the card view. In the top-right corner of your card, add a checkbox so that users can select one or more items.
+
```
Comment (afialkoff) [inline packages/v4/src/content/design-guidelines/demos/card-view/card-view.md:60]:
```suggestion

To perform a global action on one or more selected cards, users can check their desired cards' checkbox. Upon hover, a card's shadow expands, and their checkbox turns blue when selected. Once a user makes their desired card selections, they can perform global actions by clicking the global action button in the toolbar.
```

This might be made clearer by writing out the sequence vs. providing a general list of behavior. At first, I thought the numbered list indicated a procedure: But if what I'm gathering is correct, we're looking to summarize expected behavior here. Above is just a suggestion for how we can accomplish communicating the action sequence and expected behavior.

Let's also move this above the example image.

File: packages/v4/src/content/design-guidelines/demos/card-view/card-view.md [hunk 16]
```diff
@@ -0,0 +1,81 @@
...
+<img src="./img/global-actions.png" alt="card view with global action toolbar" />
+
+1. Upon hover, the card’s shadow expands.
+2. Global actions are placed in the toolbar.
+3. The card’s checkbox allows a user to perform an action on multiple cards. When a user selects a card, the checkbox turns blue to indicate selection.
+
+#### Action card
+You may use an extra-small empty state inside of a card to present the user with another means of adding a new card/item to their card view. If used, we recommend using it **in addition to** a primary button in a toolbar, to ensure that the user is still able to perform the action no matter what page of the card view they are on. The placement of the action card should be consistent with where a new card will be added, most likely the very first or the very last card in the card view.
```
Comment (afialkoff) [inline packages/v4/src/content/design-guidelines/demos/card-view/card-view.md:68]:
```suggestion
You may use an extra-small empty state inside a card to present users with another way to add more cards to the existing card view. We recommend using this feature **in addition to** a primary button in the toolbar, to ensure that the user is still able to perform the action from any page of the card view. Place the action card where a new card will be added, most likely in the very first or last card in the card view.
```

File: packages/v4/src/content/design-guidelines/demos/card-view/card-view.md [hunk 17]
```diff
@@ -0,0 +1,81 @@
...
+
+<img src="./img/add-card-empty-state.png" alt="card view with addition card" />
```
Comment (afialkoff) [inline packages/v4/src/content/design-guidelines/demos/card-view/card-view.md:70]: Same small capitalization note: "Cards" to "cards" in the image's page description.

File: packages/v4/src/content/design-guidelines/demos/card-view/card-view.md [hunk 18]
```diff
@@ -0,0 +1,81 @@
...
+
+
+### Images
+Only include meaningful images on your cards (don’t add them for decoration). Images should help users identify specific content types. Place smaller images above the card’s header, and place larger images within the card’s body.
+
+<img src="./img/card-images.png" alt="card view with logo images in each card" />
+
+### Bottom pagination
+If your card view has multiple pages, add a footer with pagination.
```
Comment (afialkoff) [inline packages/v4/src/content/design-guidelines/demos/card-view/card-view.md:79]: Can we hyperlink to pagination here?

File: packages/v4/src/content/design-guidelines/demos/card-view/card-view.md [hunk 19]
```diff
@@ -0,0 +1,81 @@
...
+
+<img src="./img/card-view-sticky-toolbar.png" alt="card view with sticky bottom toolbar" />
```
Comment (afialkoff) [inline packages/v4/src/content/design-guidelines/demos/card-view/card-view.md:81]: Same small capitalization note here: "Cards" to "cards" in the image's page description.

**Review by afialkoff — COMMENTED** [review]
Hi @mcarrano! This guideline update and demo update look great! I added comments and suggestions throughout, including some small image updates to align with our microcopy best practices. 🙂

Thanks for looping me in, and as always, take what clicks and leave what doesn't!

File: packages/v4/src/content/design-guidelines/components/card/card.md (hunk 1 above)
Comment (mcarrano) [reply packages/v4/src/content/design-guidelines/components/card/card.md:17]: I will revise these examples. Agree that the current information is confusing.

File: packages/v4/src/content/design-guidelines/demos/card-view/card-view.md (hunk 15 above)
Comment (mcarrano) [reply packages/v4/src/content/design-guidelines/demos/card-view/card-view.md:60]: I just removed this. It's not necessary.

File: packages/v4/src/content/design-guidelines/demos/card-view/card-view.md (hunk 17 above)
Comment (mcarrano) [reply packages/v4/src/content/design-guidelines/demos/card-view/card-view.md:70]: I can't find the source for this image. @mmenestr any ideas?

File: packages/v4/src/content/design-guidelines/demos/card-view/card-view.md (hunk 19 above)
Comment (mcarrano) [reply packages/v4/src/content/design-guidelines/demos/card-view/card-view.md:81]: The source for this image is missing, too!

**Comment by mcarrano** [comment]
@mmenestr @afialkoff Please take another look at this. There were a ton of updates that emerged from this review, some of which were due to prior content so I did the best I could to clear those up. Both Card and Card View guidelines have multiple changes,

@afialkoff I think I captured most of your suggested changes.

File: packages/v4/src/content/design-guidelines/components/card/card.md [hunk 20]
```diff
@@ -3,116 +3,99 @@ id: Card
...
-<br />
-<br />
-Do NOT use a card to show a dense amount of repetitive, related content. Use a list or table view instead.
+1. The first version places an image in the header with a separate title. The View details action is located in the footer.
 
+2. In the second verion, the title is moved to the header area to give a more compact footprint.
 
-## Card view usage
-A **card view** is a grid of cards that displays a small to moderate amount of content. 
+3. A third version removes the footer, adds the image back, and makes the title a link to drill into the details, which makes the footprint of the card smaller, yet.
```
Comment (mmenestr) [inline packages/v4/src/content/design-guidelines/components/card/card.md:32]: I wouldn't say "adds the image back" - I would just talk about each card as separate entities!
Suggestion: "In the third version, the card header contains an image as well as text to define the card title. The title is a link, allowing users to drill into the details of the card. This makes the footprint of the card smaller."

@afialkoff feel free to chime in on this bc I am no word smith haha

File: packages/v4/src/content/design-guidelines/components/card/card.md [hunk 21]
```diff
@@ -3,116 +3,99 @@ id: Card
...
+### Content spacing
 
-Use a card view to:
-- Display items in a data set that are best identified by a graphic or other visual representation.
-- Make information easier to visualize and compare. 
-- Summarize a large number of objects at once within the same page. 
+PatternFly supports three types of cards:
 
-Do NOT use a card view to make a large amount of text-based content easier to scan. Use a table or list view instead.
+1. **Compact (recommended):** Use 16px spacing with an 8px spacer between the header and body, and a 16px spacer between the body and footer.
```
Comment (mmenestr) [inline packages/v4/src/content/design-guidelines/components/card/card.md:38]: Would change this to say "Compact (recommended): Compact cards use 16px spacing with...." to keep the structure of the sentences in this numbered list consistent

File: packages/v4/src/content/design-guidelines/components/card/card.md [hunk 22]
```diff
@@ -3,116 +3,99 @@ id: Card
...
+2. **Standard:** Standard cards use 24px spacing with a 16px spacer between the header and body and 24px between the body and footer.
+3. **Large:** Large cards use 32px spacing with a 24px spacer between the header and the body and 32px between the body and footer. Large cards are generally used for presenting marketing content or anyplace where a lower content density is desired.
+
+<img src="./img/card-types.png" alt="compact, standard, and large card types" />
```
Comment (mmenestr) [inline packages/v4/src/content/design-guidelines/components/card/card.md:42]: Again, this image looks like it was exported at PNG 1x, could be wrong, but do you mind double checking?

File: packages/v4/src/content/design-guidelines/components/card/card.md [hunk 23]
```diff
@@ -7,112 +7,79 @@ A **card** is a square or rectangular container that can contain any kind of con
...
+### Border treatments
 
-Use a card view to:
-- Display items in a data set that are best identified by a graphic or other visual representation.
-- Make information easier to visualize and compare. 
-- Summarize a large number of objects at once within the same page. 
+PatternFly offers two types of border treatments for cards:
 
-Do NOT use a card view to make a large amount of text-based content easier to scan. Use a table or list view instead.
+1. **Default card:** By default, cards will have a thin (1px) drop shadow to differentiate them from the background of a page and give them a slightly raised appearance.
```
[the remark this answers is not in the document]
Comment (mmenestr) [reply packages/v4/src/content/design-guidelines/components/card/card.md:50]: Re-iterating^

File: packages/v4/src/content/design-guidelines/components/card/card.md [hunk 24]
```diff
@@ -7,112 +7,79 @@ A **card** is a square or rectangular container that can contain any kind of con
...
-### Card views on mobile 
-Cards will size down appropriately based on the built-in CSS responsiveness behaviors. Most of the time, they’ll follow the grid layout, expanding and compressing at certain breakpoints. At the smallest mobile size, they take up almost the entire width of the screen. 
+1. **Multi-select card:** A multi-select card has a checkbox in the upper righthand corner. Users may select cards for performing global actions against by selecting the checkbox.
 
-<img src="./img/mobile-cards.png" alt="mobile card view" width="375"/> 
+2. **Selectable card:** A selectable card has a raised appearance and a blue border treatment when selected.
 
-### Card view actions
-You can customize card actions in a few ways. Here are some recommendations to consider depending on your use case:
+<img src="./img/selectable-cards.png" alt="cards with default and flat borders " width="826" />
```
[the remark this answers is not in the document]
Comment (mmenestr) [reply packages/v4/src/content/design-guidelines/components/card/card.md:69]: Re-iterating^

File: packages/v4/src/content/design-guidelines/components/card/card.md [hunk 25]
```diff
@@ -3,116 +3,99 @@ id: Card
...
-3. The card’s checkbox allows a user to perform an action on multiple cards. When a user selects a card, the checkbox turns blue to indicate selection.
+<img src="./img/multi-select-cards.png" alt="cards in a catalog view " width="1210" />
 
-#### Action card
-You may use an extra-small empty state inside of a card to present the user with another means of adding a new card/item to their card view. If used, we recommend using it **in addition to** a primary button in a toolbar, to ensure that the user is still able to perform the action no matter what page of the card view they are on. The placement of the action card should be consistent with where a new card will be added, most likely the very first or the very last card in the card view.
 
-<img src="./img/add-card-empty-state.png" alt="card view with addition card" /> 
+### Cards in primary-detail views
 
+Use selectable cards when you want to present an overview of a set of objects and then examine the details. Here, a details of different projects can be displayed in a side-by-side primary-detail view.
```
Comment (mmenestr) [inline packages/v4/src/content/design-guidelines/components/card/card.md:93]: Would rephrase to remove the "a" in "Here, **a** details of different..." Somehow reads weird to be otherwise?

File: packages/v4/src/content/design-guidelines/demos/card-view/card-view.md [hunk 26]
```diff
@@ -0,0 +1,81 @@
...
+A **card view** is a grid of cards in a gallery to facilitate browsing. Card views are typically used to present a summary of some data set where the user can drill-down from any card to see detailed content.
+
+<img src="./img/card-view.png" alt="card view" />
+
+Use a card view to:
+- Display items in a data set that are best identified by a graphic or other visual representation.
+- Make information easier to visualize and compare.
+- Summarize a large number of objects at once within the same page.
+
+Do NOT use a card view to make a large amount of text-based content easier to scan. Use a table or list view instead.
```
[the remark this answers is not in the document]
Comment (mmenestr) [reply packages/v4/src/content/design-guidelines/demos/card-view/card-view.md:16]: This change doesn't seem to have been committed? Also, would link to table and list view guidelines here!

File: packages/v4/src/content/design-guidelines/demos/card-view/card-view.md [hunk 27]
```diff
@@ -0,0 +1,76 @@
...
+
+### Background colors for card views
+Card views can be placed on a white or a gray background, depending on your use case.
+
+**Use a white backgroun**d if your content view can change, such as when:
```
Comment (mmenestr) [inline packages/v4/src/content/design-guidelines/demos/card-view/card-view.md:21]: the d at the end of background was left out of the bolding haha

File: packages/v4/src/content/design-guidelines/demos/card-view/card-view.md [hunk 28]
```diff
@@ -0,0 +1,76 @@
...
+
+#### Inline actions
+If an action can be performed on one card only, place the action in the card’s body or footer. You don’t need to include a checkbox because the actions are card-specific.
+
+- If you have a few cards with single actions, use the [secondary button style](/components/button#variations).
+- If you have multiple cards with single actions, use a [link button](/components/button#variations).
+
+<img src="./img/few-multiple-actions.png" alt="few cards with multiple actions" />  
+
+Inline action cards are commonly used as navigation or drill-down indicators. In this case, present the user action on the card to differentiate this type of navigational card from a standard card. This way, users can easily distinguish the card types if your application includes both.
```
Comment (mmenestr) [inline packages/v4/src/content/design-guidelines/demos/card-view/card-view.md:47]: I would maybe add "In this case, present the user action on the card **as opposed to in the toolbar**" (assuming that's what this means) just to clarify a little bit. Especially since at the end it says "if your application includes both" but it's not clear what "both" are? Inline action cards and what?

File: packages/v4/src/content/design-guidelines/demos/card-view/card-view.md [hunk 29]
```diff
@@ -0,0 +1,76 @@
...
+
+
+#### Action card
+You may use an extra-small empty state inside a card to present users with another way to add more cards to the existing card view. We recommend using this feature **in addition to** a primary button in the toolbar, to ensure that the user is still able to perform the action from any page of the card view. Place the action card where a new card will be added, most likely in the very first or last card in the card view.
+
+<img src="./img/add-card-empty-state.png" alt="card view with addition card" />
+
+
+### Images
+Only include meaningful images on your cards (don’t add them for decoration). Images should help users identify specific content types. Place smaller images above the card’s header, and place larger images within the card’s body.
```
Comment (mmenestr) [inline packages/v4/src/content/design-guidelines/demos/card-view/card-view.md:69]: I'm realizing now that maybe the bullet point in the github to add another image example here was referring to adding an image to show what placing "larger images within the card's body" would look like...

**Review by mmenestr — COMMENTED** [review]
@mcarrano I added many comments - some of them are re-iterating past comments I had made that I think slipped through the cracks. Thanks so much for doing this by the way, no easy task with this much information!

File: packages/v4/src/content/design-guidelines/components/card/card.md (hunk 23 above)
[the remark this answers is not in the document]
Comment (mcarrano) [reply packages/v4/src/content/design-guidelines/components/card/card.md:50]: OK. I added this statement.

File: packages/v4/src/content/design-guidelines/components/card/card.md [hunk 30]
```diff
@@ -7,112 +7,79 @@ A **card** is a square or rectangular container that can contain any kind of con
...
-Use a white or a gray background for your card views.
+<img src="./img/card-borders.png" alt="cards with default and flat borders " width="452" />
 
-Use a white background if your content view can change. Examples: 
-- A card that opens an expandable panel on the right 
-- A card view that toggles into a table or list view 
+### Expandable cards
 
-<img src="./img/white-background.png" alt="card view with white background" /> 
+The Title element can be made expandable in situations where you want to hide the card content and reduce its footprint. In the collapsed state, only the card title is shown, and the user can click the expansion toggle to view the remainder of the card content.
```
[the remark this answers is not in the document]
Comment (mcarrano) [reply packages/v4/src/content/design-guidelines/components/card/card.md]: I changed the image above this to "Card header". I think I'm using that consistently now. Cannot find this reference to card title in the text.

File: packages/v4/src/content/design-guidelines/components/card/card.md (hunk 22 above)
Comment (mcarrano) [reply packages/v4/src/content/design-guidelines/components/card/card.md:42]: Nope. Exported at 2X.

File: packages/v4/src/content/design-guidelines/demos/card-view/card-view.md (hunk 28 above)
Comment (mcarrano) [reply packages/v4/src/content/design-guidelines/demos/card-view/card-view.md:47]: I couldn't understand what this was trying to say, so I just removed the entire paragraph. Seemed more confusing than useful.

**Comment by mcarrano** [comment]
@mmenestr I _think_ that I've addressed all of your feedback. Please see my comments and take another look.

File: packages/v4/src/content/design-guidelines/components/card/card.md [hunk 31]
```diff
@@ -3,116 +3,113 @@ id: Card
...
+3. **Body:** Provides details about the item. A card body can include any combination of static text and/or active content.
+
+4. **Footer (optional):** Contains links, actions, or static text at the bottom of a card.
+
+Cards are flexible and customizable to your use case. Here are some different ways to use a card to represent the same content. Choose a layout that considers the usage context and the space you have to work with.
+
+<img src="./img/card-variants.png" alt="three card variants" width="1202"/>
 
-<img src="./img/card.png" alt="3 parts of a card" width="394"/>  
+1. The first version places an image in the header with a separate title. The View details action is located in the footer.
```
Comment (mmenestr) [inline packages/v4/src/content/design-guidelines/components/card/card.md:28]: How do you feel about bolding the words "View details" to separate them as an action?

File: packages/v4/src/content/design-guidelines/demos/card-view/card-view.md (hunk 26 above)
[the remark this answers is not in the document]
Comment (mmenestr) [reply packages/v4/src/content/design-guidelines/demos/card-view/card-view.md:16]: @mcarrano this still doesn't appear to have gone through?

File: packages/v4/src/content/design-guidelines/demos/card-view/card-view.md [hunk 32]
```diff
@@ -0,0 +1,69 @@
...
+<img src="./img/multiple-inline.png" alt="cards with multiple inline actions in a kebab" />
+
+#### Global actions
+If an action can be performed globally on multiple cards, or if it leads to multiple pieces of content at the same time, place the action in a toolbar above the card view. In the top-right corner of your card, add a checkbox so that users can select one or more items.
+
+<img src="./img/global-actions.png" alt="card view with global action toolbar" />
+
+
+#### Action card
+You may use an extra-small empty state inside a card to present users with another way to add more cards to the existing card view. We recommend using this feature **in addition to** a primary button in the toolbar, to ensure that the user is still able to perform the action from any page of the card view. Place the action card where a new card will be added, most likely in the very first or last card in the card view.
```
Comment (mmenestr) [inline packages/v4/src/content/design-guidelines/demos/card-view/card-view.md:61]: Should the last sentence here read "most likely **as** the very first or last card in the card view" instead of "**in** the very first"?

File: packages/v4/src/content/design-guidelines/demos/card-view/card-view.md [hunk 33]
```diff
@@ -0,0 +1,69 @@
...
+
+Do NOT use a card view to make a large amount of text-based content easier to scan. Use a table or list view instead.
+
+### Background colors for card views
+Card views can be placed on a white or a gray background, depending on your use case.
+
+**Use a white background** if your content view can change, such as when:
+- A card that opens an expandable panel on the right.
+- A card view that toggles into a table or list view.
+
```
Comment (mmenestr) [inline packages/v4/src/content/design-guidelines/demos/card-view/card-view.md:24]: Adding a comment here to make sure Alana's previous comment/addition doesn't get lost.

She suggested adding "**Note**: If you use a white background, apply a gray border to your card instead of a drop shadow." 

image

File: packages/v4/src/content/design-guidelines/demos/card-view/card-view.md [hunk 34]
```diff
@@ -0,0 +1,69 @@
...
+A **card view** is a grid of cards in a gallery to facilitate browsing. Card views are typically used to present data set summaries, allowing users to drill down into any card to see more detailed content.
+
+<img src="./img/card-view.png" alt="card view" />
+
+Use a card view to:
+- Display items in a data set that are best identified by a graphic or other visual representation.
+- Make information easier to visualize and compare.
+- Summarize a large number of objects at once within the same page.
+
+Do NOT use a card view to make a large amount of text-based content easier to scan. Use a table or list view instead.
```
Comment (mmenestr) [inline packages/v4/src/content/design-guidelines/demos/card-view/card-view.md:16]: Re-commenting to make sure Alana's previous suggestion doesn't get lost since it doesn't seem to have been addressed.

Two things, change "Do NOT" to "Never" and hyperlink "table" and "list view"!

image

File: packages/v4/src/content/design-guidelines/demos/card-view/card-view.md (hunk 33 above)
Comment (mcarrano) [reply packages/v4/src/content/design-guidelines/demos/card-view/card-view.md:24]: I added a note with slightly different wording.

**Review by mmenestr — APPROVED** [review]
